int64 marker
float64[] markerincamera_measure
float64[] markerincamera_truth
float64[] eeinbase
float64[] baseincamera
float64[] markerinee
